client/src/components/UIPatterns/UIPatternsSection/UIPatternsSection.js

```diff
@@ -0,0 +1,283 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Section,
+  SectionTable,
+  DescriptionList,
+  InfiniteScrollTable,
+  InlineMessage,
+  SectionMessage
+} from '_common';
+import { useDispatch } from 'react-redux';
+import { Link, Route, Switch } from 'react-router-dom';
+
+import * as ROUTES from '../../../constants/routes';
+
+import { TicketCreateModal } from '../../Tickets';
+
+import './UIPatternsSection.module.css';
+
+const modalPath = `${ROUTES.WORKBENCH}${ROUTES.UI}/modal`;
+
+function UIPatternsSection() {
+  const dispatch = useDispatch();
+
+  function performAction() {
+    // eslint-disable-next-line no-alert
+    window.alert(
+      'Sample action (no actual action has occurred beyond this alert).'
+    );
+  }
+
+  return (
+    <dl>
+      <dt>
+        Minimal
+        <DescriptionList
+          className="small"
+          density="compact"
+          direction="horizontal"
+          data={{
+            header: 'Header',
+            content: 'Content'
+          }}
+        />
+      </dt>
+      <dd>
+        <Section header="Header" content={<p>Content</p>} />
+      </dd>
+      <dt>
+        Scrollable
+        <DescriptionList
+          className="small"
+          density="compact"
+          direction="horizontal"
+          data={{
+            contentStyleName: 'content--has-list-to-illustrate-scrolling',
+            header: 'Milk',
+            content: 'Cow, Soy, Nut',
+            contentShouldScroll: <code>true</code>
+          }}
+        />
+      </dt>
+      <dd>
+        <Section
+          contentStyleName="content--has-list-to-illustrate-scrolling"
```

Comment:
Is this property applied as a stylename to the children of DescriptionList? Is there a reason why we can't do styleName inside the child nodes?

Reply:
Answers:
- __Nope.__ That property propagates a CSS Module class name to the auto-built wrapper of the children of the `<Section … />` on which you've commented. A guide to that flow is in "Reference" → Property Flow".
- __No, but__ there is a reason that we _**should not want**_ to do `styleName` inside the child nodes: To make a standard `<Section>`—without `contentStyleName`—one must add "Much Code".

Much Code:

<details><summary>Specific Steps</summary>

1. Import `SectionContent`.
2. Add markup for `<SectionContent … >`.
3. Pass `tagName="main"`.
4. Pass `styleName="content"`.
5. Add CSS for `content`.

</details>
<details><summary>Example Code</summary>

- __JavaScript__
    ```
    import { Section, SectionContent } from '_common';
    ```
- __JSX__
    ```
    <Section header="Milk" manualContent>
      <SectionContent
          tagName="main"
          styleName="content content--has-list-to-illustrate-scrolling"
          shouldScroll
      />
        <ul>
        <li>Cow</li>
        <li>Soy</li>
        <li>Nut</li>
        </ul>
      </SectionContent>
    </Section>
    ```
- __CSS__
    ```
    .content {
        padding-left: var(--global-space--section-left);
        margin-right: var(--global-space--section-right);

        flex-grow: 1;
    }
    ```
</details>

Reference:

<details><summary>Property Flow</summary>

1. A `<Section>` (unless overridden via `<Section manualContent />` prop) creates `<SectionContent>`.
1. The `<Section contentStyleName />` prop is a [manual alias](https://github.com/TACC/Core-Portal/blob/task/FP-385-section-comp-welcome-msg--sections/client/.babelrc#L23) to propagate CSS Module class to `<Section contentClassName />` prop.
1. A `<SectionContent styleName />` prop is an [automatic alias](https://github.com/gajus/babel-plugin-react-css-modules#anonymous-stylename-resolution) to propagate CSS Module class to `<SectionContent className />` prop.
1. A `<SectionContent className />` prop applies a CSS class to the `<SectionContent>` root element.

</details>

Notes:

The `<Section>` can automatically create `<SectionHeader>`, `<SectionContent>`, and `<WelcomeMessage>`. But a dev may also create those manually (via `manualHeader`, `manualContent`, and `messages` or `welcomeMessageText`). The manual creation is intended to be reserved for edge cases, not to be explicit (because the verbosity and maintenance for consistency is high).

Reply:
Okay I think I was getting thrown by 

```
<Section contentStyleName="">
<SectionContent styleName="">
```